src/SamplesApp/Directory.Build.props

```diff
@@ -7,7 +7,11 @@
     <!-- fixed for self-testing inside the Uno Solution -->
     <UnoRemoteControlPort>53487</UnoRemoteControlPort>
   </PropertyGroup>
-
+  
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+  </ItemGroup>
```

Comment:
We're generally updating the versions in `src/Directory.Build.props`, why did you need it scoped to the samples app? Is the updated version only used by the sample app, or exposed through Uno ?

Reply:
I only saw that the samples app was referencing Newtonsoft, so didn't push to the parent Directory.Build.props.